src/azure-cli/azure/cli/command_modules/vm/_validators.py

```diff
@@ -1284,6 +1284,15 @@ def _validate_vm_vmss_msi(cmd, namespace, is_identity_assign=False):
     elif namespace.identity_scope or namespace.identity_role:
         raise ArgumentUsageError('usage error: --assign-identity [--scope SCOPE] [--role ROLE]')
 
+    # Enable system assigned msi by default when Trusted Launch configuration is met
+    if hasattr(namespace, 'security_type') and hasattr(namespace, 'enable_vtpm') and\
+            hasattr(namespace, 'enable_secure_boot'):
+        if namespace.security_type and namespace.enable_vtpm and namespace.enable_secure_boot:
```

Comment:
We need to judge whether the value of `securityType` is `TrustedLaunch` to confirm whether the Trusted Launch configuration is met

Reply:
@sirfamelin Could you please confirm whether we need to cancel enabling System MSI when the user passes in parameter `--disable-integrity-monitoring`?

Reply:
Please add the info level log to inform users that the MSI is enabled by default when Trusted Launch configuration is met

Reply:
Yes, MSI does not need to be enabled when the --disable-integrity-monitoring parameter is passed in. However, if the user enables MSI from another command, that should supersede the --disable-integrity-monitoring parameter as the user might have another need for MSI that is separate from this scope.